sentinel-core/transport/command/BUILD

```diff
@@ -0,0 +1,21 @@
+load("//bazel:copts.bzl", "DEFAULT_COPTS", "TEST_COPTS")
+
+cc_library(
+    name = "transport_httpserver",
```

Comment:
Please split this library, this library is too big.the purpose of doing this is to make our dependence clearer.